pkg/controller/clusterdeployment/clusterdeployment_deleted.go

```diff
@@ -29,7 +29,7 @@ import (
 func (r *ReconcileClusterDeployment) handleDelete(request reconcile.Request, instance *hivev1.ClusterDeployment) (reconcile.Result, error) {
 	r.reqLogger.Info("Deleting syncset")
 
-	ssName := fmt.Sprintf("%v-pd-sync", instance.Name)
+	ssName := instance.Name + "-pd-sync"
```

Comment:
Why switch this?

Reply:
After reading it all over it was just easier for me to read